compile-run-with-classpath/solution.sh

```diff
@@ -7,3 +7,5 @@
 # First argument is blank: true
 # Second argument is blank: true
 # Third argument is blank: false
+javac -cp commons-lang3-3.9.jar StringIsBlank.java
+java -cp commons-lang3-3.9.jar;. StringIsBlank '' ' ' '*'
```

Comment:
```suggestion
java -cp commons-lang3-3.9.jar:. StringIsBlank '' ' ' '*'
```

在Unix系统上是冒号，在Windows系统上才是分号。